test/envtest/reconciler_setupwithmanager_test.go

```diff
@@ -42,24 +46,42 @@ func TestNewKonnectEntityReconciler(t *testing.T) {
 type konnectEntityReconcilerTestCase struct {
 	name                string
 	objectOps           func(ctx context.Context, t *testing.T, cl client.Client, ns *corev1.Namespace)
+	mockSDKSettings     func(t *testing.T, m *ops.MockSDKWrapper, ns *corev1.Namespace)
```

Comment:
WDYT about naming this `expectations` or `mockExpectations`? Settings doesn't fit well here IMHO.

Reply:
renamed it to `mockExpectations`.